toc/working-groups/WORKING-GROUPS.md

```diff
@@ -221,6 +222,27 @@ Mission: Provides discreet management of security vulnerabilities issues relevan
 | -------------------------------------------------------- | ---------------- | ------- | --------------------------------------- |
 | <img width="30px" src="https://github.com/paulcwarren.png"> | Paul Warren       | VMWare  | [@paulcwarren](https://github.com/paulcwarren) |
 
+
+## Concourse
+
+Mission: Provides the Concourse CI/CD platform.
+
+This working group has no repositories in the `cloudfoundry` GitHub organization.
+
+| Artifact                   | Link |
+| -------------------------- | ---- |
+| Charter                    | [concourse.md](./concourse.md) |
+| Forum                      | TBD  |
```

Comment:
```suggestion
| Forum                      | [GitHub Discussions](https://github.com/concourse/concourse/discussions)  |
```

There was a Discourse forum, but that's gone. This is the only repo we have Discussions enabled. Not sure if it's suitable for the Forum field here, just a suggestion.